llvm/test/CodeGen/PowerPC/f128-bitcast.ll

```diff
@@ -86,3 +86,25 @@ entry:
   ret i64 %1
 }
 
+define <4 x i32> @test(i512 %a) {
```

Comment:
nit: Can we maybe be a bit more descriptive for the test title? Not sure if something like `truncBitcast` makes sense, or something like that?